.circleci/config.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         type: boolean
     environment:
       TEST_RESULTS: /tmp/test-results
-      DKTL_VERSION: "4.2.9"
```

Comment:
What's with the "RC"? "Release Candidate" maybe?

Reply:
@clayliddell yes I am testing the changes on [DKAN-Tools](https://github.com/GetDKAN/dkan-tools/pull/328)